lib/heroku/command/pg.rb

```diff
@@ -14,6 +14,16 @@ def set_commands(shorthand)
     end
   end
 
+  # pg:stats_reset
+  # calls the Postgres functions pg_stat_reset()
```

Comment:
Should we add more documentation here? This is pretty thin, and since there's already the similarly-named `pg_stat_statements_reset` function, this might be confusing without more info...